src/Forms.js

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from "react";
+import "./Forms.css";
+
+const Forms = ({ onClose, onSubmit, formType, content, setContent }) => {
+  const handleChange = useCallback((event) => setContent(event.target.value), [
+    setContent,
+  ]);
+  const handleSubmit = useCallback(
+    (e) => {
+      e.preventDefault();
+      onSubmit(content);
+      onClose(e);
```

Comment:
Why pass `e` to `onClose`?

Reply:
The close button uses the same `onClose` function to handle its shutdown, but when in the form, it wants to try to submit so in the higher level function I added prevent default, but then in handle submit, it needed the `e` too..
would it be better to make a function for `Forms` and then handle default there, and call `onClose`?